torch/utils/benchmark/utils/compile.py

```diff
@@ -0,0 +1,192 @@
+import torch
+
+__all__ = ["bench_all", "benchmark_compile"]
+
+import torch
+import torch._dynamo as torchdynamo
+import time
+import warnings
+from typing import Optional, List
+
+# Added this otherwise all the warnings from each backend will be printed
+# Not sure if this warning will propgate if user is not using this config
+warnings.filterwarnings("ignore")
```

Comment:
No, you can't do this. This will clobber whatever user warning setting on import, which is a big no no for libraries. If you really really want to suppress the warnings, do it with a context manager in the utilities (but it's not obvious to me that you want to suppress warnings?)